packages/cli/src/commands/upgrade.js

```diff
@@ -0,0 +1,50 @@
+import execa from 'execa'
+import Listr from 'listr'
+
+import c from 'src/lib/colors'
+
+export const command = 'upgrade'
+export const desc = 'Upgrade all @redwoodjs packages via interactive CLI'
+
+export const builder = (yargs) => {
+  yargs
+    .option('check', {
+      alias: 'c',
+      type: 'boolean',
+      default: false,
+      description: 'Check for outdated packages without upgrading',
+    })
+    .strict()
+}
+
+const rwPackages =
+  '@redwoodjs/core @redwoodjs/api @redwoodjs/web @redwoodjs/router'
+
+export const handler = async ({ check }) => {
+  const tasks = new Listr([
+    {
+      title: "Running 'redwood upgrade'",
+      task: (_ctx, task) => {
+        if (check) {
+          task.title = 'Checking available upgrades for @redwoodjs packages'
+          execa(`yarn outdated ${rwPackages}`, undefined, {
```

Comment:
```suggestion
          execa.command(`yarn outdated ${rwPackages}`, {
```
Use [`execa.command`](https://github.com/sindresorhus/execa#execacommandcommand-options) to specify both file & arguments in single string.